src/main/java/com/github/fabienrenaud/jjb/model/Users.java

```diff
@@ -140,6 +148,182 @@ public int hashCode() {
         public String toString() {
             return "JsonDataObj{" + "_id=" + _id + ", index=" + index + ", guid=" + guid + ", isActive=" + isActive + ", balance=" + balance + ", picture=" + picture + ", age=" + age + ", eyeColor=" + eyeColor + ", name=" + name + ", gender=" + gender + ", company=" + company + ", email=" + email + ", phone=" + phone + ", address=" + address + ", about=" + about + ", registered=" + registered + ", latitude=" + latitude + ", longitude=" + longitude + ", tags=" + tags + ", friends=" + friends + ", greeting=" + greeting + ", favoriteFruit=" + favoriteFruit + '}';
         }
+
+        public String get_id() {
+            return _id;
+        }
+
+        public void set_id(String _id) {
+            this._id = _id;
+        }
```

Comment:
Can we rename these `getId` and `setId` (and do that in Clients.java too)? I don't recall why I put an underscore prefix to the `id` attribute. Best guess: it was to work around some limitation or bug from a lib... but hopefully that isn't an issue with the getters/setters.